### ISSUE TITLE
Deleting Entries is not accessible

### ISSUE THREAD

**Issue by heull001** [freeotp/freeotp-ios#307]
Sweeping entries to remove them is not accessible for screenreader-users. There should be an additional method. E.g. an edit-button which opens a menue where you can mark entries and delete them, or a delete-button for every entry.

For testing, you can enable the iOS-screenreader VoiceOver in settings -> accessibility.
